daml-lf/kv-support/src/main/scala/com/daml/lf/kv/archives/RawArchive.scala

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.kv.archives
+
+import com.google.protobuf.ByteString
+
+case class RawArchive(byteString: ByteString) extends AnyVal
```

Comment:
I think a doc comment that explains the protobuf message in the bytestring would be good.